03week/towersOfHanoi.js

```diff
@@ -19,32 +19,95 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+function movePiece(first, second) {
+  
+  let move = stacks[first].pop()
+  // console.log(move + "1")
+  count++
+  // console.log(stacks);
+  stacks[second].push(move)
+  console.log("You've made " + count + " moves so far")
+  checkForWin()
+  // getPrompt();
+  
+//   endStack.push(move);
+//   console.log(endStack);
 }
 
-function isLegal() {
-  // Your code here
-
+function isLegal(first, second) {
```

Comment:
You program runs, but it is not split up correctly.
isLegal is doing too much by calling move(), and prompt. Its job should be to return true or false.

The towersOfHanoi() method is the method that should handle your control flow.